specs/latest/2.0/index.html

```diff
@@ -785,27 +785,29 @@ <h2 class="no-toc">Table of contents</h2>
                             GLsizei height, GLint border, GLintptr offset);
   void compressedTexImage2D(GLenum target, GLint level, GLenum internalformat, GLsizei width,
                             GLsizei height, GLint border, ArrayBufferView srcData,
-                            optional GLuint srcOffset = 0);
+                            optional GLuint srcOffset = 0, optional GLuint srcLengthOverride = 0);
```

Comment:
Note: having just implemented this in Chromium -- both of these situations need to generate INVALID_VALUE in order to make the test above pass.